src/util/saveArticles.ts

```diff
@@ -412,13 +412,26 @@ export async function treatVideo(mw: MediaWiki, dump: Dump, srcCache: KVS<boolea
     }
 
     videoSources = videoSources.sort(widthXHeightSorter);
+    const videoElArea = (videoEl.getAttribute('height') || 0) * (videoEl.getAttribute('width') || 0);
```

Comment:
I don't understand really the computation with width*heigth. Considering that all the videos have the same ratio (width/height), I think just comparing the widths should be enough.